resources/js/app.ts

```diff
@@ -1,8 +1,8 @@
-import Alpine from 'alpinejs';
-import focus from '@alpinejs/focus';
 // eslint-disable-next-line camelcase,import/no-unresolved
 // import { livewire_hot_reload } from 'virtual:livewire-hot-reload';
 
+// eslint-disable-next-line import/no-unresolved
+import { Livewire } from 'livewire/livewire.esm';
```

Comment:
The TypeScript language server used by VSCode is throwing a type error here. I'm not sure why it's not showing up in the build itself, but I was able to resolve it locally with the following steps:

1. Adjust the alias to point at the ESM bundle:
```ts
        livewire: resolve(__dirname, './vendor/livewire/livewire/dist/livewire.esm'),
```

2. In _resources/js_ (the same directory as _app.ts_), create a new file called _livewire.d.ts_ with the following content:

```ts
declare module 'livewire' {
  export const Livewire: any;
}
```

3. In _app.ts_, adjust the import:
```ts
import { Livewire } from 'livewire';
```

Reply:
awesome, thanks! done in https://github.com/RetroAchievements/RAWeb/pull/2061/commits/e6af867b2d71338a286b4946329759bae914afc5